src/CheckIfDead.php

```diff
@@ -271,8 +298,8 @@ protected function processCurlResults( $curlInfo, $full = false ) {
 		}
 		// Check for error messages in redirected URL string
 		if ( strpos( $effectiveUrlClean, '404.htm' ) !== false ||
-			strpos( $effectiveUrlClean, '/404/' ) !== false ||
-			stripos( $effectiveUrlClean, 'notfound' ) !== false
+		     strpos( $effectiveUrlClean, '/404/' ) !== false ||
+		     stripos( $effectiveUrlClean, 'notfound' ) !== false
```

Comment:
Have you checked the file itself.  It looks pretty nice, but I can change it back too.